HelpSource/Classes/Server.schelp

```diff
@@ -623,6 +623,13 @@ an optional instance of link::Classes/Condition:: used for evaluating this.
 argument:: ... args
 one or more valid OSC messages.
 
+method:: isLocal
+Returns true if current server is local
```

Comment:
i removed the code example. an experienced supercollider user won't actually need much explanation in order to make something like this